src/averageAges.js

```diff
@@ -20,8 +20,15 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
-}
+  const men = people.filter(({ sex }) => sex === 'm');
+  const menCentury = men.filter(({ died }) => {
+    return Math.ceil(died / 100) === century;
+  });
+  const filteredMen = century !== undefined ? menCentury : men;
```

Comment:
The century must be a number, so let's check if century is a number, we use menCentury else - men

Reply:
Fix code style here and in the whole file, please)